Core.lua

```diff
@@ -1192,28 +1177,638 @@ function Core:AppendTeam(teamData)
+		-- //STUB TD - Nav Container
+		teamNavContainer = GOW.GUI:Create("InlineGroup");
+		teamNavContainer:SetWidth(250);
+		teamNavContainer:SetLayout("Flow");
+		teamNavContainer:SetFullHeight(true);
 
-	containerScrollFrame:AddChild(itemGroup);
+		if GoWTeamTabContainer then
+			GoWTeamTabContainer:AddChild(teamNavContainer);
+		end
+
+		-- //STUB TD - Information Container
+		teamInfoContainer = GOW.GUI:Create("InlineGroup");
+		teamInfoContainer:SetLayout("Flow");
+		teamInfoContainer:SetWidth(710);
+		teamInfoContainer:SetFullHeight(true);
+		teamInfoContainer:SetPoint("TOP", GoWTeamTabContainer.frame, "TOP", 0, 0);
+
+		if GoWTeamTabContainer then
+			GoWTeamTabContainer:AddChild(teamInfoContainer);
+		end
+
+		-- //!SECTION
+
+		-- //SECTION - TD - Summary
+		-- //STUB Team URL
+		local teamURL = GOW.GUI:Create("SFX-Info-URL");
+		teamURL:SetLabel("Team Link");
+		teamURL:SetText(teamData.webUrl);
+		teamURL:SetDisabled(false);
+		teamInfoContainer:AddChild(teamURL);
+
+		-- //STUB Team Description
+		local teamDescriptionLabel = GOW.GUI:Create("SFX-Info");
+		teamDescriptionLabel:SetLabel("Description");
+		if teamData.description == "" then
+			teamDescriptionLabel:SetText("No description provided.")
+		else
+			teamDescriptionLabel:SetText(teamData.description);
+			teamDescriptionLabel:SetCallback("OnEnter", function(self)
+				local tooltip = LibQTip:Acquire("TeamDescriptionTooltip", 1, "LEFT");
+				GOW.tooltip = tooltip;
+
+				tooltip:AddHeader('|cffffcc00Team Description');
+				local line = tooltip:AddLine();
+				tooltip:SetCell(line, 1, teamData.description, "LEFT", 1, nil, 0, 0, 300, 50);
+				tooltip:SmartAnchorTo(self.frame);
+				tooltip:Show();
+			end);
+			teamDescriptionLabel:SetCallback("OnLeave", function()
+				LibQTip:Release(GOW.tooltip);
+				GOW.tooltip = nil;
+			end);
+		end
+		teamDescriptionLabel:SetDisabled(false);
+
+		if teamInfoContainer then
+			teamInfoContainer:AddChild(teamDescriptionLabel);
+		end
+
+		-- // STUB Hide Offline Members Button
+		local HideOfflineMembersButton = GOW.GUI:Create("CheckBox")
+		HideOfflineMembersButton:SetLabel("Hide Offline Members")
+		HideOfflineMembersButton:SetValue(isOfflineChecked)
+		HideOfflineMembersButton:SetType("checkbox")
+		HideOfflineMembersButton:SetDisabled(false)
+		HideOfflineMembersButton:SetCallback("OnValueChanged", function()
+			if GoWScrollTeamMemberContainer then
+				if GoWTeamMemberContainer then
+					GoWTeamMemberContainer:ReleaseChildren()
+				end
+				local teamGroup = GoWScrollTeamMemberContainer:GetUserData("teamGroup") -- used to get the current teamGroup selected from the navigation buttons
+				local filterValue = currentFilterValue
+				local checkBoxValue = HideOfflineMembersButton:GetValue()
+				if checkBoxValue then
+					-- args: teamGroup, hideOffline, specRole
+					Core:RenderFilteredTeamMembers(teamGroup, true, filterValue)
+				else
+					Core:RenderFilteredTeamMembers(teamGroup, false, filterValue)
+				end
```

Comment:
Can be simplified
Core:RenderFilteredTeamMembers(teamGroup, checkBoxValue, filterValue)